MREUnityRuntime/MREUnityRuntimeLib/Core/ActorManager.cs

```diff
@@ -101,10 +109,16 @@ internal void ProcessActorCommand(Guid actorId, NetworkCommandPayload payload, A
             queue.Enqueue(payload, onCompleteCallback);
         }
 
+        private List<ActorCommandQueue> _queuesForUpdate = new List<ActorCommandQueue>();
+
```

Comment:
Why is this a class member, instead of just being inside update()? It seems to be only used inside update(), and cleared at start of each update?

Reply:
An optimization. I didn't want to reallocate it every frame.

---
In reply to: [263543869](https://github.com/Microsoft/mixed-reality-extension-unity/pull/48#discussion_r263543869) [](ancestors = 263543869)